docker-compose-generator/docker-fragments/bitcoin-eclair.yml

```diff
@@ -0,0 +1,55 @@
+version: "3"
+
+services:
+  bitcoind:
+    environment:
+      BITCOIN_NETWORK: ${NBITCOIN_NETWORK:-regtest}
+      BITCOIN_EXTRA_ARGS: |
+        zmqpubrawblock=tcp://0.0.0.0:28332
+        zmqpubrawtx=tcp://0.0.0.0:28333
+        deprecatedrpc=signrawtransaction
+        addresstype=p2sh-segwit
+        rpcuser=ceiwHEbqWI83
+        rpcpassword=DwubwWsoo3
+  eclair_bitcoin:
+    restart: unless-stopped
+    stop_signal: SIGINT
```

Comment:
is it really needed? Can you try if eclair close nicely without it?

Reply:
we dont need it, just copy pasta

Reply:
tested. We actually do need it.

Reply:
Are you sure? It should exit gracefully?

Reply:
ah nvm, this was with my old custom image before

Reply:
Actually you need SIGKILL.

Reply:
@araspitzu told me to use SIGINT instead of SIGKILL if anything

Reply:
SIGINT does terminate eclair and it's more graceful than SIGKILL, consider that it *should* also be safe to kill it as well but it's bad practice.